postgres/suite.go

```diff
@@ -0,0 +1,55 @@
+package postgres
+
+import (
+	"database/sql"
+
+	driverSql "github.com/go-sql-driver/mysql"
```

Comment:
is it the right choice? I'm afraid this is not the right one.

How about this: https://github.com/lib/pq??